src/test/java/net/pms/dlna/DLNAMediaInfoTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * PS3 Media Server, for streaming any medias to your PS3.
+ * Copyright (C) 2012  I. Sokolov
```

Comment:
@Sami32 Hehe, yes and no. I took an existing test class and duplicated and modified because I didn't want to have to set up a test class "from scratch" in case I forgot somthing.